docs/upgrades/unreleased.md

```diff
@@ -47,6 +47,44 @@
             }
         ```
 
+* Update jss-translation-client-loader service to get the performance improvement during fetching Dictionary Data Fetching for SSR
```

Comment:
Can you, please, check the .md file in "preview"? since some statements are not aligned